Vagrantfile

```diff
@@ -1,6 +1,53 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+default_os = 'ubuntu'
+ENV['OS'] ||= default_os
+
+#
+# CoreOS
+#
+Vagrant.configure('2') do |config|
+  # Vagrant plugins
+  %w(vagrant-ignition).each do |name|
+    next if Vagrant.has_plugin? name
+    Dir.chdir(Dir.home) { system "vagrant plugin install #{name}" }
+  end
+
+  update_channel = 'alpha'
+  config.vm.box = "coreos-#{update_channel}"
+  config.vm.box_url = "https://#{update_channel}.release.core-os.net/amd64-usr/current/coreos_production_vagrant_virtualbox.json"
+  config.ssh.insert_key = false
+  config.ssh.forward_agent = true
+
+  # Masters and Workers
+  (1..6).each do |i|
+    name = (1..3).include?(i) ?  "master-#{i}" : "worker-#{i - 3}"
+    config.vm.define name, autostart: %w(1 2 3).include?("#{i}") do |c|
+      c.vm.provider 'virtualbox' do |v|
+        v.check_guest_additions = false
+        v.functional_vboxsf = false
+        v.memory = 2048
+        v.cpus = 1
```

Comment:
CoreOS does not start when the number of CPUs is set to 2. 🤔  So skip validation for kubeadm. https://github.com/linyows/k8v/pull/3/files#diff-303cb7a1491ffb7b36e1aff65bcceaffR45